product_template_tags/views/product_template_tag.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_tag_form_view">
+        <field name="name">product.template.tag.form (in product_template_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <form>
+                <header/>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="%(product.product_template_action)d" type="action"
+                                class="oe_stat_button" icon="fa-archive"
+                                context="{'search_default_tag_ids': active_id}">
+                            <field name="products_count" widget="statinfo"/>
+                        </button>
+                    </div>
+
+                    <group>
+                        <field name="name"/>
+                        <field name="company_id" widget="selection" groups="base.group_multi_company"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="product_template_tag_search_view">
+        <field name="name">product.template.tag.search (in product_template_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="company_id"/>
+                <group expand="0">
+                    <filter name="group_by_company" string="Company"
+                            domain="[]" context="{'group_by': 'company_id'}"
+                            groups="base.group_multi_company"/>
+                </group>
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="product_template_tag_tree_view">
+        <field name="name">product.template.tag.tree (in product_template_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+                <field name="company_id" widget="selection" groups="base.group_multi_company"/>
+                <field name="products_count"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="product_template_tag_act_window">
+        <field name="name">Product Template Tag</field>
+        <field name="res_model">product.template.tag</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_type">form</field>
+    </record>
+
+    <record model="ir.ui.menu" id="product_template_tag_config_menu">
```

Comment:
Maybe this would make more sense under Stock -> Products -> Configuration? But of course then you would need stock. Would it be a bad idea to set it as a dependency? Probably. Might as well keep the menu here, still it seems a little bit out of place.